notifications.bs

```diff
@@ -253,15 +277,23 @@ section of HTML. [[!HTML]]
 
 <p>User agents are expected to honor the Unicode semantics of the text of a
 <a lt="concept notification">notification</a>'s
-<a lt="concept title">title</a> and <a>body</a>. Each is expected to be
-treated as an independent set of one or more bidirectional algorithm paragraphs
-when displayed, as defined by the bidirectional algorithm's rules P1, P2, and
-P3, including, for instance, supporting the paragraph-breaking behavior of
-U+000A LINE FEED (LF) characters. For each paragraph of the
-<a lt="concept title">title</a> and <a>body</a>, the
+<a lt="concept title">title</a>, <a>body</a>, and the
+{{NotificationAction/title}} of each of its <a>actions</a>. Each is expected to
```

Comment:
Done, though note that it made things much more verbose, for example rather than:
```html
  <li><p>Set <var>notification</var>'s list of <a>actions</a> to the first
  <var>n</var> entries from <var>options</var>'s <code>actions</code>, where
  <var>n</var> is the <a>maximum number of actions</a> supported.
```
I ended up with:
```html
  <li><p>Set <var>notification</var>'s list of <a>actions</a> to an empty list,
  then for each <var>entry</var> in <var>options</var>'s <code>actions</code>,
  up to the <a>maximum number of actions</a> supported (skip any excess
  entries), perform the following steps:

  <ol>
    <li><p>Let <var>action</var> be a new <a lt="actions">action</a>.

    <li><p>Set <var>action</var>'s <a>action name</a> to the <var>entry</var>'s
    <code>action</code>.

    <li><p>Set <var>action</var>'s <a lt="action title">title</a> to the
    <var>entry</var>'s <code>title</code>.

    <li><p>Append <var>action</var> to <var>notification</var>'s list of
    <a>actions</a>.
  </ol>
```
And similarly for the conversion back from concept to `NotificationAction`.